src/repositories/hive/hive.repository.mock.ts

```diff
@@ -79,7 +79,7 @@ export class MockHiveRepository extends HiveRepository {
   async getAccount(author: string) {
     const [hiveAccount] = await this._hive.database.getAccounts([author]);
     if (process.env.TEST_PUBLIC_KEY)
-      hiveAccount.posting.key_auths.push([process.env.TEST_PUBLIC_KEY, 0]);
+      hiveAccount.posting.key_auths.push([process.env.TEST_PUBLIC_KEY, 1]);
```

Comment:
This might need to be formatted, but this will ensure success even if the hive account is configured with a non-standard threshold

```suggestion
      hiveAccount.posting.key_auths.push([process.env.TEST_PUBLIC_KEY, hiveAccount.posting.weight_threshold]);
```